### DISCUSSION TITLE
Clearing the data

### DISCUSSION THREAD

**Discussion by viris** [Security-Onion-Solutions/securityonion#11976] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

64

### Storage for /

1 TB

### Storage for /nsm

1TB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I wanted to clear nsm and elastic data.
Donet as suggested here:
https://github.com/Security-Onion-Solutions/securityonion/discussions/5839

After that, no more data is coming to elastic.

so-status shows no error.
Tried to find, out what could be done, but no clue. Reinstalling is the only way.

Tried this 2 times.

Any clues whay clear tools are not working?

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
I just tried running both so-nsm-clear and so-elastic-clear and then verified that new data is coming into Elastic as expected.

Check the logs in /opt/so/log/ for additional clues as to why it's not working for you.
